metadata.json

```diff
@@ -15,26 +15,30 @@
       "operatingsystem": "RedHat",
       "operatingsystemrelease": [
         "7",
-        "8"
+        "8",
+        "9"
       ]
     },
     {
       "operatingsystem": "CentOS",
       "operatingsystemrelease": [
         "7",
-        "8"
+        "8",
```

Comment:
Would this be a good time to remove EOL CentOS 7 and 8?

Reply:
removal goes into a separate PR usually